kotlinpoet/src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -884,4 +884,13 @@ class KotlinPoetTest {
       |}
       |""".trimMargin())
   }
+
+  // https://github.com/square/kotlinpoet/issues/701
+  @Test fun noIllegalCharacterInIdentifier() {
+    assertThrows<java.lang.IllegalArgumentException> {
```

Comment:
nit: Remove `java.lang.`.

Reply:
Not sure how `java.lang` got in there in the first place. Same above